asaman/__init__.py

```diff
@@ -21,8 +21,7 @@
 os.umask(0o022)
 
 # The build directory value
-# TODO: This should be updated based on Windows or Linux
-WHEEL_BUILD_DIR = "/tmp/pip-wheel-build"
+WHEEL_BUILD_DIR = os.path.join(tempfile.gettempdir(), "pip-wheel-build")
```

Comment:
For reproducibility we have to make sure that the path is pre-defined in our project level, there is a reason I decided not to use tempfile module in this case. 

Also did you build wheels on windows to verify if they are reproducible or not? What all environment values can cause a break in that in Windows?